integration-tests/pom.xml

```diff
@@ -24,6 +24,23 @@
       <version>${quarkus.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.couchbase.client</groupId>
+      <artifactId>metrics-micrometer</artifactId>
+      <version>0.7.6</version>
```

Comment:
In future maybe we should look at using a BOM, so we don't have to specify (and maintain) multiple versions.